### PR TITLE
Making changes to test common CI indexers

### PR DESCRIPTION
**Pull request by vishnuchalla**
### Description
Test PR to do a reharse and test perf-scale common [CI indexers](https://github.com/vishnuchalla/e2e-benchmarking/blob/vchalla/workloads/kube-burner-ocp-wrapper/index.sh).

### CODE REVIEW

**Comment by vishnuchalla** [comment]
@rook @paigerube14 PTAL 👋🏼

**Comment by paigerube14** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

File: ci-operator/step-registry/openshift-qe/node-density/openshift-qe-node-density-commands.sh [hunk 1]
```diff
@@ -13,13 +13,21 @@ source ./venv_qe/bin/activate
...
 export EXTRA_FLAGS="--pods-per-node=$PODS_PER_NODE --pod-ready-threshold=$POD_READY_THRESHOLD"
 export WORKLOAD=node-density
 
 export ES_SERVER="https://$ES_USERNAME:$ES_PASSWORD@search-ocp-qe-perf-scale-test-elk-hcm7wtsqpxy7xogbu72bor4uve.us-east-1.es.amazonaws.com"
 
 rm -rf "${SHARED_DIR}/${OUTPUT_FILE:?}"
+PROW_JOB_START=$(date +"%Y-%m-%d %H:%M:%S")
 ./run.sh |& tee "${SHARED_DIR}/${OUTPUT_FILE}"
+PROW_JOB_END=$(date +"%Y-%m-%d %H:%M:%S")
+PROW_JOB_STATUS="success"
```
[the remark this answers is not in the document]
Comment (vishnuchalla) [reply ci-operator/step-registry/openshift-qe/node-density/openshift-qe-node-density-commands.sh]: Added to check based on command exit code to determine SUCCESS/FAILURE. Thank you!

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by jtaleric** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

**Comment by vishnuchalla** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-gcp-4.13-ocp-qe-perfscale-ci-tests-kube-burner

File: Makefile [hunk 2]
```diff
@@ -2,12 +2,12 @@ SHELL=/usr/bin/env bash -o errexit
 
 .PHONY: help check check-boskos check-core check-services dry-core core dry-services services all update template-allowlist release-controllers checkconfig jobs ci-operator-config registry-metadata boskos-config prow-config validate-step-registry new-repo branch-cut prow-config multi-arch-gen
 
-export CONTAINER_ENGINE ?= docker
+export CONTAINER_ENGINE ?= podman
```
Comment (paigerube14) [inline Makefile:5]: we should move this to a separate pr so that it could get proper review on this change from its owners

**Comment by vishnuchalla** [comment]
Thanks to this PR as it helped me to test my changes [here](https://github.com/cloud-bulldozer/e2e-benchmarking/pull/597). Closing it!
